src/Nancy/AfterPipeline.cs

```diff
@@ -5,43 +5,108 @@ namespace Nancy
     using System.Threading.Tasks;
     using Nancy.Helpers;
 
+    /// <summary>
+    /// Intercepts the request after the appropriate route handler has completed its operation.
+    /// The After hooks does not have any return value because one has already been produced by the appropriate route. 
+    /// Instead you get the option to modify (or completely replace) the existing response by accessing the Response property of the NancyContext that is passed in.
+    /// </summary>
+    /// <seealso>

```

Comment:
Is this syntax correct? Looks like it should be `<seealso cref="...."/> https://msdn.microsoft.com/en-us/library/xhd7ehkk.aspx